internal/controllers/clusterrequest_controller_test.go

```diff
@@ -544,27 +544,39 @@ var _ = Describe("ClusterRequestReconcile", func() {
 				},
 			},
 		}
-
+		var (
+			testFullTemplateSchema = fmt.Sprintf(`{
```

Comment:
Since it's being used in multiple places, could we just define this var globally outside of this unittest ?

Reply:
Fixed